consensus/src/state_computer.rs

```diff
@@ -287,6 +296,7 @@ impl StateComputer for ExecutionProxy {
             .as_ref()
             .cloned()
             .expect("must be set within an epoch");
+        let mut commit_futures = FuturesUnordered::new();
```

Comment:
don't see why we need to be fancy here, in fact we need to ensure the commits happen in sequence. 
would just use a vec and wait on them one after another.

Reply:
yeah I wanted to use FuturesOrdered too

Reply:
-_- not even that, no need to pull them in concurrency IIUC, just a Vec would do.